help/sources/tutorials/ui/create/marketing-automation/braze.md

```diff
@@ -0,0 +1,58 @@
+---
+keywords: Experience Platform;home;popular topics;braze;Braze,Currents
+solution: Experience Platform
+title: Create a Braze Source Connection in the UI
+type: Tutorial
+description: Learn how to create a Braze source connection using the Adobe Experience Platform UI and Braze Currents
+exl-id: 452b7290-b9e8-4728-8b58-0e0c76bd9449
+---
+# Create a [!DNL Braze] source connection in the UI
+
+Source connectors in Adobe Experience Platform provide the ability to ingest externally sourced data on a scheduled basis. This tutorial provides steps for creating a streaming [!DNL Braze] source connector using the [!DNL Platform] user interface.
+
```

Comment:
Can we add one or two lines here to talk about what Braze is and maybe a use case example on what customers can use it for? Something like: "Braze is... Use the Braze source to bring {what kinds of data?} from your Braze account to Adobe Experience Platform."

Reply:
Great feedback, will add that in.